index.js

```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
 // Read the history file and parse the data
 
 function readFile() {
-  const array = fs.readFileSync("testHistory.txt").toString().split("\n");
+  const homeDir = os.homedir();
+  const historyPath = path.join(homeDir, ".bash_history");
+  const possiblePath = [
+    path.join(homeDir, ".bash_history"),
+    path.join(homeDir, ".zsh_history"),
+    path.join(homeDir, ".history"),
+  ];
 
-  return array;
+  // need to find the biggest history file
+
+  let biggestHistorySize = 0;
+  let biggestHistoryPath;
```

Comment:
Great work here, maybe we should move the const to the top of the file? Maybe better for organization. Also, maybe we should export the functions later for other developers by `export default function`